pattern-library/sass/patterns/_forms.scss

```diff
@@ -260,12 +260,15 @@ $input-width: (
 
     // Input fields immediately followed by a button (ie, search boxes) should eliminate the spacing
     // between them and adopt matching styling/sizing to emphasize that they are connected.
-    &[type=search] + button {
+    + button {
         border-top-left-radius: 0;
         border-bottom-left-radius: 0;
         font-size: font-size(base);
         margin-left: (-1 * $baseline / 4);
     }
+    &::-webkit-search-cancel-button {
+        display: none;
```

Comment:
ok, in that case I would handle removing it in `_normalize.scss` as well. Should just be a straight revert of when I enabled it here: https://github.com/edx/ux-pattern-library/commit/422d73da24810c6804a05f0ec8355da33c197893#diff-2afbb5f9793b6fe59d7da2d2a4b28beaL358